modules/react/layout/lib/Layout.tsx

```diff
@@ -81,8 +81,8 @@ class Layout extends React.Component<LayoutProps> {
 
   public render() {
     const {
-      gutter = canvas.spacing.xs,
-      spacing = spacingNumbers.xs,
+      gutter = canvas.space.xs,
```

Comment:
It's nice that it caught this. Properties require a separate check than Member Expressions.